sdk/textanalytics/ai-text-analytics/test/apiKey.spec.ts

```diff
@@ -77,4 +77,449 @@ describe("[API Key] TextAnalyticsClient", function() {
     assert.equal(results.length, 1);
     assertAllSuccess(results);
   });
+
+  describe("#health", () => {
```

Comment:
feels kinda bad this file is named `apiKey.spec.ts` but it's not really about testing API keys, it's about testing client methods.

Reply:
Yeah, I was actually thinking of having a single test suite parameterized over the method of authentication and I created this as a first step: https://github.com/Azure/azure-sdk-for-js/pull/11046